### PR TITLE
Move the s2i-dpdk build code to this repo

### PR DESCRIPTION
**Pull request by SchSeba**
This PR move ths s2i-dpdk from https://github.com/SchSeba/s2i-dpdk-centos7 to this repo.

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

**Comment by slintes** [comment]
Do you mind adding a REAMDE to tools/s2i-dpdk, explaining what the purpose of the image is?
At least I have no clue ;)
Thanks!

**Comment by SchSeba** [comment]
@slintes done I hope this is more clear now :)

File: tools/s2i-dpdk/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+This folder also contains a simple dpdk workload test inside the `test/test-app` folder.
+The test application use `expect` to run `testpmd`.
+
+# How to use
+
+To use this s2i you only need to run the [build-config.yaml](base-image/build-config.yaml).
+
+This will create a new "dpdk-testing" namespace then it will configure a "ImageStream" for the image and start a build. 
+
+After the build is ready you can edit the dpdk demo folder under the feature-configs.
```
Comment (fedepaol) [inline tools/s2i-dpdk/README.md]: Can we suggest to build another layer instead of editing a file?

File: tools/s2i-dpdk/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# dpdk-centos7
+FROM openshift/base-centos7
+#FROM centos:7
+
+LABEL maintainer="Sebastian Scheinkman <sebassch@gmail.com>"
```
Comment (fedepaol) [inline tools/s2i-dpdk/Dockerfile]: Does it make sense to have your rh address instead of gmail one?

Comment (SchSeba) [reply tools/s2i-dpdk/Dockerfile]: I prefer to have this one here. This is the one I am using for github also

File: tools/s2i-dpdk/README.md (hunk 1 above)
Comment (SchSeba) [reply tools/s2i-dpdk/README.md]: Done :)

**Pull request merged**